src/Traversals/depthfirst.jl

```diff
@@ -2,13 +2,11 @@ import Base.showerror
 struct CycleError <: Exception end
 Base.showerror(io::IO, e::CycleError) = print(io, "Cycles are not allowed in this function.")
 
-
-
 struct DepthFirst{F<:Function} <: TraversalAlgorithm
     neighborfn::F
 end
 
-DepthFirst(;neighborfn=outneighbors) = DepthFirst(neighborfn)
+DepthFirst(;neighborfn = outneighbors) = DepthFirst(neighborfn)
```

Comment:
we don't put spaces between assignments in signatures (this is the exception to the general rule).

Reply:
done. thanks